src/instructlab/model/train.py

```diff
@@ -206,6 +206,18 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     required=True,  # default from config
     config_sections=ADDITIONAL_ARGUMENTS,
 )
+@click.option(
+    "--fsdp-cpu-offload-optimizer",
+    type=bool,
+    cls=clickext.ConfigOption,
+    help="Allow CPU offload for FSDP optimizer.",
+)
+@click.option(
+    "--fsdp-sharding-strategy",
+    type=str,
+    cls=clickext.ConfigOption,
+    help="harding strategy to use for FSDP optimizer.",
```

Comment:
ditto

Reply:
per @Maxusmusti we should not be exposing sharding strategy right now